src/V3Width.cpp

```diff
@@ -3524,6 +3526,37 @@ class WidthVisitor final : public VNVisitor {
                                                << "() should be handled");
         }
     }
+    void methodCallIfaceRef(AstMethodCall* nodep, AstIfaceRefDType* adtypep) {
+        AstIface* const ifacep = adtypep->ifacep();
+        UINFO(1, __FUNCTION__ << ":" << nodep << endl);
+        if (AstNodeFTask* const ftaskp
+            = VN_CAST(m_memberMap.findMember(ifacep, nodep->name()), NodeFTask)) {
```

Comment:
Can this fail?  I think if e.g. referencing something that isn't a NodeFTask say a constraint from an interface?  If so, please add a test for the error and appropriate _bad.out file to check the error.

Reply:
That sounds like a possibility. On it, thanks.

Reply:
While adding these tests I decided to add an explicit error message for this case also, as the errors gave no useful information to user.